eureka/disable_test.go

```diff
@@ -0,0 +1,57 @@
+package eureka_test
+
+import (
+	"github.com/pivotal-cf/spring-cloud-services-cli-plugin/eureka"
+
+	"errors"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/pivotal-cf/spring-cloud-services-cli-plugin/httpclient/httpclientfakes"
+)
+
+var _ = Describe("Disable", func() {
+	const testAccessToken = "someaccesstoken"
+
+	var (
+		fakeAuthClient *httpclientfakes.FakeAuthenticatedClient
+		eurekaUrl      string
+		eurekaAppName  string
+		instanceId     string
+
+		err error
+	)
+
+	BeforeEach(func() {
+		fakeAuthClient = &httpclientfakes.FakeAuthenticatedClient{}
+		eurekaUrl = "http://some.host/x/y/cli/instances/someguid"
+		eurekaAppName = "eureakappname"
+		instanceId = "instanceid"
+
+	})
+
+	JustBeforeEach(func() {
+		err = eureka.Disable(fakeAuthClient, eurekaUrl, eurekaAppName, instanceId, testAccessToken)
+	})
+
+	It("should issue a PUT with the correct parameters", func() {
+		Expect(fakeAuthClient.DoAuthenticatedPutCallCount()).To(Equal(1))
+		url, accessToken := fakeAuthClient.DoAuthenticatedPutArgsForCall(0)
+		Expect(url).To(Equal("http://some.host/x/y/cli/instances/someguideureka/apps/eureakappname/instanceid/status?value=OUT_OF_SERVICE"))
+		Expect(accessToken).To(Equal(testAccessToken))
+		Expect(err).NotTo(HaveOccurred())
+	})
+
+	Context("when DELETE return an error", func() {
```

Comment:
Should be PUT rather than DELETE in the description text?

Reply:
@georgeharley Good eyes! Fixed.